cli/src/registry/publish.ts

```diff
@@ -170,6 +183,26 @@ command.action(async options => {
         'WasmModule' in foundEntry.value && foundEntry.value.WasmModule.version === entry.version,
     )?.id;
 
+    const moduleHashBuffer = createHash('sha256').update(entry.module).digest();
+    const moduleHashHex = moduleHashBuffer.toString('hex');
+    const moduleUint8Array = new Uint8Array(moduleHashBuffer);
+
+    const moduleStoredFileName =
+      `${entry.name}_${entry.version}_${moduleHashHex.substring(0, 8)}`.replace(
+        /[^a-zA-Z0-9_]/g,
+        '',
```

Comment:
why this replace needed?

Reply:
i was not sure if the asset canister would have restrictions for the filenames

Reply:
Where does the name come from? I mean: why would it have nonstandard characters at the first place?

Reply:
the non standard characters come from the package name, which we publish to the registry as `@orbit/station` or `@orbit/upgrader`, and that would also break the file name due to `/`